src/bin/tq.rs

```diff
@@ -132,11 +136,13 @@ fn main() -> anyhow::Result<()> {
                     .print()?;
             }
         }
+    } else {
+        println!("{}", format!("{}", output).trim_matches('"'));
     }
 
     // If there is not syntax highlighting, just print normally.
     #[cfg(not(feature = "syntax-highlighting"))]
-    println!("{output}");
+    println!("{}", format!("{}", output).trim_matches('"'));
```

Comment:
```suggestion
    if !app.raw {
        println!("{output}");
    } else {
        println!("{}", output.trim_matches('"'));
    }
right? Otherwise `--raw` becomes redundant by default
```